### PR TITLE
Update pypi-publish.yml

### PR DESCRIPTION
**Pull request by ori-shalom**

### CODE REVIEW

File: .github/workflows/pypi-publish.yml [hunk 1]
```diff
@@ -66,14 +62,13 @@ jobs:
...
       - name: Run example
-        if: success()
         run: |
           poetry install
           PVAULT_SERVICE_LICENSE=${{ secrets.PVAULT_SERVICE_LICENSE }} /bin/bash run.sh
         working-directory: examples/django-encryption-example
 
       - name: Build and publish
-        if: success() && inputs.skip_publish != 'true'
+        if: inputs.bump-package != 'none'
```
Comment (david-piiano) [inline .github/workflows/pypi-publish.yml:73]: Why did you remove `success()`

**Review by david-piiano — APPROVED** [review]
Looks good - left one question to answer before merge

File: .github/workflows/pypi-publish.yml (hunk 1 above)
Comment (ori-shalom) [reply .github/workflows/pypi-publish.yml:73]: Because this is the default behavior.
Jobs stops on the first failed step unless mentioned otherwise
